scripts/ilks2ascii.sh

```diff
@@ -1,11 +1,19 @@
-#!/bin/bash
-### ilks2ascii -- list ilks from ilk-registry in ascii
-### Usage: ./ilks2ascii <ilk-registry-address>
+#!/usr/bin/env bash
+### ilks2ascii -- list ilks from the IlkRegistry in ascii
+### Usage: ./ilks2ascii.sh
 
-ILKS=$(seth call "$1" "list()(bytes32[])")
+set -e
 
-for ilk in $(echo "$ILKS" | sed "s/,/ /g")
-do
-	seth --to-ascii "$ilk"
+[[ "$ETH_RPC_URL" ]] || { echo "Please set a ETH_RPC_URL"; exit 1; }
+
+CHANGELOG=0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F
+
+ILK_REGISTRY=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "ILK_REGISTRY")")")
```

Comment:
Can you update this script to use foundry?

Reply:
I've tried replacing `seth` with `cast` but doesn't seem to work out-of-the-box, we could migrate to foundry in a separate pr eventually, to reproduce and address those issues.